third_party/npm_core-js/README.chromium

```diff
@@ -0,0 +1,4 @@
+Name: core-js
```

Comment:
all of our deps in package.json must define the license file, which I believe gets automatically included in `chrome://credits`. Otherwise the build fails.